src/FakeTime/FakeTime.h

```diff
@@ -16,6 +16,12 @@ class FakeTime {
   FakeTime(int h, int m, int s);
 
   int formatTime(char *targetBuffer);
+
+  /**
+   * Updates the Fake time so that it show the hour, minute and second
+   * in future (after given milliseconds value).
+   */
+  void plusMillis(long millis);
```

Comment:
What should this function do?
If its time deviation from rtc, shouldn't it be like void FakeTime(int h, int m, int s, int deviation,bool changeFromRtc)? As every time we run FakeTime we should know:
1. If the time was deviated from RTC,
2. If yes, what is the deviation?